packages/components/textarea/src/textarea.stories.ts

```diff
@@ -134,68 +130,6 @@ export const Label: StoryObj = {
   }
 };
 
-export const Hint: StoryObj = {
-  render: () => {
-    return html`
-      <style>
-        form {
-          display: flex;
-          flex-direction: column;
-        }
-
-        sl-textarea {
-          margin-bottom: 1rem;
-        }
-      </style>
-      <form>
-        ${hintSizes.map((hintSize, id) => {
-          return html`
-            <sl-label for="form-textarea-${id}">Nickname</sl-label>
-            <sl-textarea
-              id="form-textarea-${id}"
-              hint="What would you like people to call you?"
-              hintSize=${hintSize}
-            ></sl-textarea>
-          `;
-        })}
-        <sl-label for="textarea4">Nickname</sl-label>
-        <sl-textarea
-          id="textarea4"
-          disabled
-          hint="What would you like people to call you?"
-          hintSize="lg"
-          value="Disabled textarea"
-        ></sl-textarea>
-      </form>
-    `;
-  }
-};
-
-export const RichLabelHint: StoryObj = {
-  render: () => html`
-    <style>
-      form {
-        display: flex;
-        flex-direction: column;
-      }
-
-      div {
-        gap: 0.25rem;
-      }
-    </style>
-    <form>
-      <sl-label for="textarea">
-        <label slot="label">Custom <i>label</i></label>
-      </sl-label>
-      <sl-textarea id="textarea">
-        <div slot="hint">
-          Hint is an accessible way to provide <strong>additional information</strong> that might help the user
-        </div>
-      </sl-textarea>
-    </form>
-  `
-};
-
 export const MinMaxLength: StoryObj = {
```

Comment:
Can you please update the example?
<img width="666" alt="image" src="https://github.com/sl-design-system/components/assets/111562742/c1b3a4c8-e99b-4511-8f75-4f1db61c9b6d">

When the textarea is invalid it should have red border and red icon etc.

Reply:
In this example as well :)

<img width="628" alt="image" src="https://github.com/sl-design-system/components/assets/111562742/476929cc-748a-4245-9878-50d6f685f0e1">